### PR TITLE
Remove redundant activation events from samples

### PR DESCRIPTION
**Pull request by joyceerhl**

### CODE REVIEW

File: .base-sample/package.json [hunk 1]
```diff
@@ -11,14 +11,12 @@
 		"url": "https://github.com/Microsoft/vscode-extension-samples"
 	},
 	"engines": {
-		"vscode": "^1.32.0"
+		"vscode": "^1.74-insider"
```
Comment (Tyriar) [inline .base-sample/package.json:14]: We should do this until we release 1.74

**Review by Tyriar — DISMISSED** [review]

**Review by mjbvz — CHANGES_REQUESTED** [review]
Change LGTM but please hold off on checking in this change until 1.74 is released (and switch the targets to normal 1.74 instead of insiders)

**Comment by TylerLeonhardt** [comment]
The github auth sample could be included in this :)

File: github-authentication-sample/package.json [hunk 2]
```diff
@@ -16,9 +16,7 @@
 	"categories": [
 		"Other"
 	],
-	"activationEvents": [
-		"onCommand:extension.getGitHubUser"
-	],
+	"activationEvents": [],
```
Comment (TylerLeonhardt) [inline github-authentication-sample/package.json:19]: need engine change?

**Review by TylerLeonhardt — DISMISSED** [review]
maybe you also want to bump the types in these extensions? Not sure if it's needed though.

**Comment by joyceerhl** [comment]
AFAICT bumping the types is not required. vsce only complains if you try to use types from a newer version of VS Code than is required in the engine property.
https://insiders.vscode.dev/github/microsoft/vscode-vsce/blob/1936e8b56627e873ebffa2390e4d1a466676cc65/src/validation.ts#L99-L101

**Review by mjbvz — APPROVED** [review]

**Comment by broken-bytes** [comment]
The custom editor sample does not work anymore. 

Setup:
image

**Comment by joyceerhl** [comment]
@broken-bytes The custom editor sample works as expected for me:

image


If it still doesn't work for you, could you please file a new issue with reproducible steps so we can investigate further? Thanks!

**Pull request merged**